src/main/java/com/pingcap/tikv/expression/aggregate/AggregationFunction.java

```diff
@@ -0,0 +1,6 @@
+package com.pingcap.tikv.expression.aggregate;
```

Comment:
What's the purpose of this empty class? And, no copyright info.

Reply:
My bad, class removed.